instrumentation/opentelemetry-instrumentation-urllib3/setup.py

```diff
@@ -11,6 +11,12 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+
+# DO NOT EDIT. THIS FILE WAS AUTOGENERATED FROM templaes/urllib3.
```

Comment:
should this say it was generated from instrumentation_setup.py?

Reply:
Yes. This is incorrect. Fixing.